R/package-auth.R

```diff
@@ -46,8 +47,8 @@ db_host <- function(id = NULL, prefix = NULL) {
 #' @import cli
 #' @export
 db_token <- function() {
-  token <- Sys.getenv("DATABRICKS_TOKEN")
-
+  #token <- Sys.getenv("DATABRICKS_TOKEN")
```

Comment:
Same expectation as commented around `db_host()`.

Reply:
added env argument defaulting to the option'db_env'. If NULL db_get_parm will use DEFAULT in both cases (.databrickscfg and .Renviron) which is explaiend in below comment